cache/edge_lambdas/src/redirects.ts

```diff
@@ -19,193 +19,103 @@
  *
  */
 export const literalRedirects: Record<string, string> = {
-  '/event-series/Wo1YeCoAACoAZFoN': '/events/Wqkd1yUAAB8sW4By',
-  '/explore': '/stories',
+  '/access': '/pages/accessibility',
+  '/alice-anderson':
+    '/exhibitions/alice-anderson--memory-movement-memory-objects',
+  '/articles/aids-posters-0': '/articles/aids-posters',
+  '/articles/hysteria': '/articles/what-is-hysteria', // "hysteria" already taken by ZN-ELxEAACMABO5a in former UID.
+  '/articles/lustmord': '/articles/WqfvayUAAKsrVreh', // lustmord-and-the-three-perspectives-of-murder -> not published?
```

Comment:
[Waiting to hear back from Editorial in Slack](https://wellcome.slack.com/archives/C3N7J05TK/p1729602171817339)

Reply:
While waiting to hear back, I've changed it to point to the UID, simplifies it.